.github/workflows/cron.yaml

```diff
@@ -3,8 +3,9 @@ name: Test dask-upstream
 
 on:
   schedule:
+    # 06:15 UTC daily.
     # 18:15 UTC daily.
-    # We want to run after the nightly pipeline finishes.
+    # We at least one one run after the nightly pipeline finishes.
     # https://github.com/rapidsai/workflows/blob/main/.github/workflows/nightly-pipeline-trigger.yaml is
     # currently set to 5:00 UTC and takes ~12 hours
     - cron: "15 18 * * *"
```

Comment:
Does this line need to be updated (or do we need to add a new line for 6:15 UTC)?

Reply:
This might be an older commit (apologies if I copy-pasted the wrong link). As of https://github.com/rapidsai/dask-upstream-testing/pull/13/files#diff-9b920fe58586f44bdeac3b93c4e54812b8af360283f70e288dbbc7461c8b2825R10 things should be good.